### PR TITLE
ci: fix async loss and speed

### PR DESCRIPTION
**Pull request by wangraying**

### CODE REVIEW

File: .buildkite/scripts/benchmark_master.sh [hunk 1]
```diff
@@ -50,9 +77,9 @@ python -m bagua.distributed.launch \
     ${COMMUNICATION_SCRIPT}
 
 SYNTHETIC_SCRIPT="/workdir/examples/benchmark/synthetic_benchmark.py"
-algorithms=(gradient_allreduce bytegrad decentralized low_precision_decentralized qadam)
-speeds=(185.0 180.0 150.0 115.0 170 0)
-losses=(0.001763 0.001694 0.002583 0.001821 0.000010 0.00000)
+algorithms=(gradient_allreduce bytegrad decentralized low_precision_decentralized qadam async)
+speeds=(185.0 180.0 150.0 115.0 170 200)
+losses=(0.001763 0.001694 0.002583 0.001821 0.000010 0.004000)
```
Comment (wangraying) [inline .buildkite/scripts/benchmark_master.sh:82]: These two values remain in the air

**Review by NOBLES5E — CHANGES_REQUESTED** [review]
gpu test seems crashed

**Comment by wangraying** [comment]
> gpu test seems crashed

bagua-core needs to merge first

**Comment by NOBLES5E** [comment]
> > gpu test seems crashed
> 
> bagua-core needs to merge first

Done. Try triggering test again :)

**Comment by wangraying** [comment]
merge to https://github.com/BaguaSys/bagua/pull/185

**Pull request closed without merging**